packages/api/webpack.config.js

```diff
@@ -81,6 +81,7 @@ module.exports = {
   },
   externals: [
     'aws-sdk',
+    '@aws-sdk',
```

Comment:
Is the `@aws-sdk` the v3 version of `aws-sdk`? Adding to `externals` reduces our webpack size somehow?

Reply:
It does, in that we're not including it as a top level dependency.   As noted yesterday, this needs removed until we upgrade to node v18